setup.py

```diff
@@ -0,0 +1,55 @@
+# flake8: noqa
+from setuptools import setup
+
+
+# File used to allow install from a branch
+# ex: !pip3 install --upgrade git+https://github.com/zillow/datasets.git@tz/plugins
+
+# To create this file
+# poetry build --format sdist
+# tar -xvf dist/*-`poetry version -s`.tar.gz -O '*/setup.py' > setup.py
+
+
+packages = [
+    "datasets",
+    "datasets.plugins",
+]
+
+package_data = {"": ["*"]}
+
+install_requires = [
+    "importlib-metadata>=4.8.1,<5.0.0",
+    "pandas>=1.1.0,<2.0.0",
+    "pyarrow>=5.0.0,<6.0.0",
+    "click>=7.0,<8",
+]
+
+extras_require = {
+    ':extra == "dask"': ["dask>=2021.9.1,<2022.0.0"],
+    ':extra == "spark"': ["pyspark>=3.0.0,<4.0.0"],
+    ':extra == "metaflow"': ["zillow-metaflow @ " "git+https://github.com/zillow/metaflow.git@tz/coverage"],
+}
+
+entry_points = {
+    "datasets.executors": ["metaflow_executor = " "datasets.plugins:MetaflowExecutor"],
```

Comment:
Did you mean to add in the extra `" "` here and on lines 30 & 37?

Reply:
`black` did this!

Reply:
Weird, cool will resolve then as not too fussed and would prefer just to conform 😅